> $ nix-build -A pkgsCross.aarch64-multiplatform.gitFull
> make: Entering directory '/build/git-2.50.1/contrib/credential/libsecret'
> /nix/store/gkwbw9nzbkbz298njbn3577zmrnglbbi-bash-5.3p0/bin/bash: line 1: pkg-config: command not found
> /nix/store/gkwbw9nzbkbz298njbn3577zmrnglbbi-bash-5.3p0/bin/bash: line 1: pkg-config: command not found
> aarch64-unknown-linux-gnu-gcc -g -O2   -o git-credential-libsecret.o -c git-credential-libsecret.c
> git-credential-libsecret.c:29:10: fatal error: glib.h: No such file or directory
>    29 | #include <glib.h>
>       |          ^~~~~~~~
> compilation terminated.
> make: *** [Makefile:22: git-credential-libsecret.o] Error 1
> make: Leaving directory '/build/git-2.50.1/contrib/credential/libsecret'
> error: build of '/nix/store/795445jzkw1ilfwhscwvn30klndb39cf-git-aarch64-unknown-linux-gnu-2.50.1.drv' failed with exit code 2;
> ```

if i apply this patch on top of your PR, then `pkgsCross.aarch64-multiplatform.gitFull` builds!

```diff
diff --git pkgs/applications/version-management/git/default.nix pkgs/applications/version-management/git/default.nix
index 93452cac87f2..67e7e9924c10 100644
--- pkgs/applications/version-management/git/default.nix
+++ pkgs/applications/version-management/git/default.nix
@@ -115,6 +115,8 @@ stdenv.mkDerivation (finalAttrs: {
     # Fix references to gettext introduced by ./git-sh-i18n.patch
     substituteInPlace git-sh-i18n.sh \
         --subst-var-by gettext ${gettext}
+    substituteInPlace contrib/credential/libsecret/Makefile \
+        --replace-fail 'pkg-config' "$PKG_CONFIG"
   ''
   + lib.optionalString doInstallCheck ''
     # ensure we are using the correct shell when executing the test scripts
```

--- a/pkgs/applications/version-management/git/default.nix
+++ b/pkgs/applications/version-management/git/default.nix
@@ -115,6 +115,8 @@ stdenv.mkDerivation (finalAttrs: {
     # Fix references to gettext introduced by ./git-sh-i18n.patch
     substituteInPlace git-sh-i18n.sh \
         --subst-var-by gettext ${gettext}
+    substituteInPlace contrib/credential/libsecret/Makefile \
+        --replace-fail 'pkg-config' "$PKG_CONFIG"
   ''
   + lib.optionalString doInstallCheck ''
     # ensure we are using the correct shell when executing the test scripts

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -1202,9 +1202,9 @@ with pkgs;
 
   # The full-featured Git.
   gitFull = git.override {
-    svnSupport = true;
+    svnSupport = stdenv.buildPlatform == stdenv.hostPlatform;
     guiSupport = true;
-    sendEmailSupport = true;
+    sendEmailSupport = stdenv.buildPlatform == stdenv.hostPlatform;
     withSsh = true;
     withLibsecret = !stdenv.hostPlatform.isDarwin;
   };

